easybuild/tools/filetools.py

```diff
@@ -670,7 +670,7 @@ def det_patched_files(path=None, txt=None, omit_ab_prefix=False, github=False, f
     :param filter_deleted: filter out all files that were deleted by the patch
     """
     if github:
-        patched_regex = r"^diff --git (?P<ab_prefix>[ab]/)?(?P<file>\S+)"
+        patched_regex = r"^diff --git (?:[ab]/)?\S+\s*(?P<ab_prefix>[ab]/)?(?P<file>\S+)"
```

Comment:
OK

I'm not sure how to deal with paths that include spaces here, so I'll just make a clear mention of that in the docstring (pretty sure we won't be needing that anytime soon, since we don't include spaces in names).